scripts/static-vdis

```diff
@@ -301,7 +303,7 @@ def attach(vdi_uuid):
                     os.unlink(d + "/disk")
                 except:
                     pass
-                path = None
+                path = ""
```

Comment:
The empty string is not a valid path, why is this substitution valid?

Reply:
If the following `if` statements do not set a valid `path`, this function (`attach`) would currently raise a `NameError` because the variable would be unbound in this case:
```py
os.symlink(path, d + "/disk")
```
As I think we should keep the current behaviour and add should issue better diagnostics, I added a comment and added raising a `NameError` like it would be raised currently.

Please re-review, to check if that looks better to you.